ext/standard/tests/streams/bug51056.phpt

```diff
@@ -0,0 +1,44 @@
+--TEST--
+Bug #51056 (fread() on blocking stream will block even if data is available)
+--SKIPIF--
```

Comment:
As of PHP 8.1.0, all possibly shared extensions required by a test should be listed in its `--EXTENSION--` section (run-tests.php will try to load them, if not already loaded):
```suggestion
--EXTENSIONS--
pcntl
--SKIPIF--
```
Although I would prefer a test which doesn't need `pcntl_fork()` so it could run on Windows as well.

Reply:
Yeah good point, I just rewrote the test to ServerClientTestCase so hopefully will work on windows.